apigov-23206 - Adding the team exclusion (#177)
Axway/amplify-central: content/en/docs/manage_marketplace/set_product_visibility.md

> 2. Select **Products > All Products**. The product list view is displayed.

Reviewer: ```suggestion
2. Select **All Products**. The product list view is displayed.
```

3. Select a product to view its details.
4. Click **Set Visibility**.
5. Under **Platform Users**, select one of the following options from the **Visible To** menu:

>     * **Everyone** - (default) make the product visible to all registered users in your provider organization.

Reviewer: ```suggestion
    * **Everyone** - (default) all registered users in your provider organization can see the product in the Marketplace
```

    * **Selected teams** - only members of the selected teams can see the product in the Marketplace

>     * **Exclude selected teams** - only members not part of selected teams can see the product in Marketplace

Reviewer: ```suggestion
    * **Exclude selected teams** - only members that are not part of selected teams can see the product in the Marketplace
```

>     * **None** - nobody on arketpalce will be able to see the product

Reviewer: ```suggestion
    * **None** - the product is not visible to anyone in the Marketplace
```
